### DISCUSSION TITLE
请问一下 ws+cdn 被墙时候的表现

### DISCUSSION THREAD

**Discussion by ghost** [v2fly/v2ray-core#2025] [Q&A]
我 https 的网站可以用本地网络正常进去。
ping 域名和 IP 都能 ping 通。
22 端口也没被封。
但是今早突然不能用了，后面换成 CF 的CDN 又可以上了。配置完全没变，但用 IP 会连不上。

我不是很懂。但是这个、应该也不是被墙了吧？

**Comment by Yochee** [comment]
套cdn就是换ip
你用原ip连等于没套

**Reply by ghost** [reply]
原來用tls，但也沒看被牆就不能用了
